Makefile

```diff
@@ -69,6 +69,12 @@ reset-net: check-deploy-env
 		reset-net.yml
 
 mnesia_snapshot:
+ifeq ($(BACKUP_DB_VERSION),)
```

Comment:
maybe we should combine BACKUP_DB_VERSION and DEPLOY_DB_VERSION in some more generic name and use in both cases

Reply:
@dincho sure propose generic name i will change it...

Reply:
Just a suggestion, not for this PR

Reply:
Ok gotcha